Lib9c/Action/MintAssets.cs

```diff
@@ -69,8 +84,33 @@ public override IAccount Execute(IActionContext context)
                     }
 
                     state = state.SetState(inventoryAddr, inventory.Serialize());
+                    fivs.Add(itemsNotNull);
                 }
+            }
 
+            foreach (var recipient in mailRecords.Keys)
+            {
+                if (
+                    state.GetState(recipient) is Dictionary dict &&
+                    dict.TryGetValue((Text)SerializeKeys.MailBoxKey, out IValue rawMailBox)
+                )
+                {
+                    var mailBox = new MailBox((List)rawMailBox);
+                    (List<FungibleAssetValue> favs, List<FungibleItemValue> fivs) = mailRecords[recipient];
+                    mailBox.Add(
+                        new UnloadFromMyGaragesRecipientMail(
+                            context.BlockIndex,
+                            context.GetRandom().GenerateRandomGuid(),
```

Comment:
I think it should use only one random instance per one action context. Or maybe the mail id will be all the same.

https://github.com/planetarium/libplanet/blob/e555adddeac03765f7a4efd6015fb53cbb5231b7/Libplanet.Action/ActionContext.cs#L70-L71

Reply:
Amended. https://github.com/planetarium/lib9c/pull/2237/commits/8a5d8b431675752181724a0420ea66caa7e0c03c